pdns/dnsdist-web.cc

```diff
@@ -443,6 +443,22 @@ static void connectionThread(int sock, ComboAddress remote)
           output << "\n";
         }
 
+        // Latency histogram buckets
+        output << "# HELP dnsdist_latency_bucket Number of queries contributing to response time histogram per latency bucket\n";
+        output << "# TYPE dnsdist_latency_bucket counter\n";
```

Comment:
A Prometheus histogram definition should look like this:

```
# HELP django_http_requests_body_total_bytes Histogram of requests by body size.
# TYPE django_http_requests_body_total_bytes histogram
django_http_requests_body_total_bytes_bucket{le="0.0"} 31794.0
django_http_requests_body_total_bytes_bucket{le="1.0"} 31794.0
(...)
django_http_requests_body_total_bytes_bucket{le="+Inf"} 31897.0
django_http_requests_body_total_bytes_count 31897.0
django_http_requests_body_total_bytes_sum 11935.0
```

Note that the metric name does not include `_bucket`. The type is `histogram`, not `counter` (the sum and count are also part of this metric type).

Reply:
I now included a histogram type as well. As some of the metrics already get their help and type from pre-existing code I've simply added a histogram type now. Hope that works as expected.